include/zephyr/dt-bindings/input/input.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022, tangchunhui@coros.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_H_
+
+/* keyboard code */
+
+#define KEY_CODE_RESERVED   0
```

Comment:
The current code set may be fine for a basic sample program, but definitely won't allow a complex system that covers the full range of possible keyboard codes, other possible layouts/languages, etc.